python/ray/tune/trial_runner.py

```
@@ -132,13 +132,21 @@ def __init__(self,
                  fail_fast=False,
                  verbose=True,
                  checkpoint_period=None,
-                 trial_executor=None):
+                 trial_executor=None,
+                 metric=None):
         self._search_alg = search_alg or BasicVariantGenerator()
         self._scheduler_alg = scheduler or FIFOScheduler()
         self.trial_executor = trial_executor or RayTrialExecutor()
 
+        self._metric = metric
+
         # For debugging, it may be useful to halt trials after some time has
         # elapsed. TODO(ekl) consider exposing this in the API.
+        if "TRIALRUNNER_WALLTIME_LIMIT" in os.environ:
+            logger.warning(
+                "The TRIALRUNNER_WALLTIME_LIMIT environment variable is "
+                "deprecated and will be removed in the future. "
+                "Use `tune.run(time_budget_s=limit)` instead.")
```

I would just raise an error for this one